src/Axe/Models/CodeBlock.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Axe.Models
+{
+    /// <summary>
+    /// Class represents code block which is used in Roslyn platform compiler
+    /// </summary>
+    public class CodeBlock
+    {
+        /// <summary>
+        /// Identifier
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Source code
+        /// </summary>
+        public string Code { get; set; }
+
+        public string Task { get; set; }
+
+        /// <summary>
+        /// Returned result of source code 
+        /// </summary>
+        public string Output { get; set; }
```

Comment:
needs discussion - why do we need `Output`? in my understanding `Code` is self-contained, it declares all data it needs for work. (I judge by how dotnetfiddle works). Controller executes codeand sends all results (compilation errors or output) to user. User can change some data, execute again and get different results.

Reply:
You're right, but it's necessary to check result of user. By assignment: user read the task (input and output (Output field) data is known), write code, then system executes code and inserts 'input' data to code and checks output. If the output is right the task is solved.